### ISSUE TITLE
[Q} pyqt4

### ISSUE THREAD

**Issue by ildar** [flathub/com.riverbankcomputing.PyQt.BaseApp#231]
1. anyone having the BaseApp for pyqt4?
2. Would be nice to have a couple of good advises how to build that. I'm a total newb in flatpaks.

Thanks in advance.

**Comment by JakobDev** [comment]
No, PyQt4 is outdated since many Years, so there will be no BaseApp. I also can't help you with building, as the buildsystem may have changed over the Years. Try to look at old build scripts for PyQt4 to see how it works.

**Comment by ildar** [comment]
On Tue, Oct 17, 2023 at 12:57 PM JakobDev wrote:

> Try to look at old build scripts for PyQt4 to see how it works.
>
you mean the raw PyQt4 building or (hopefully) some vintage PyQt4 flatpak
out of my radares?

thanks for answering!

**Comment by JakobDev** [comment]
I don't think threes a Flatpak out there, but you can maybe take a lock at other Packages of PyQt4 to sse how it's been build in General.

**Comment by hfiguiere** [comment]
this is a wontfix. Build Qt4.

We even removed it from shared-modules 

https://github.com/flathub/shared-modules/pull/287
